portfolio/src/main/webapp/sheetsApi.js

```diff
@@ -147,16 +147,13 @@ function singleScheduledTaskToSheetsArray(scheduledTask) {
   const task = scheduledTask.task;
   const taskName = task.name;
   // Changes seconds into minutes.
-  const taskDurationMinutes = task.duration.seconds / 60;
-  const taskDescription = task.description.value;
-  const taskPriority = task.priority.priority;
-  const taskTimeSeconds = scheduledTask.startTime.seconds;
-  const taskDate = new Date();
-  // This is x1000 because the functions takes milliseconds
-  taskDate.setTime(taskTimeSeconds * 1000);
+  const taskDurationMinutes = task.duration;
```

Comment:
(Optional) I think task.duration should be task.durationMinutes instead. Feel free to add TODO comment if you want to make that change later.

Reply:
I will leave a TODO.